src/selectize.js

```diff
@@ -1091,7 +1091,8 @@ $.extend(Selectize.prototype, {
 		}
 
 		// perform search
-		if (query !== self.lastQuery) {
+    if (query !== self.lastQuery) {
+      if (settings.normalize) query = query.normalize("NFD").replace(/[\u0300-\u036f]/g, "");
```

Comment:
what does normalize do?   and what is the "NFD" value refer to?  are there any other options?

Reply:
Sorry, I thought I sent this commit to another PR. it removes all accents in the string.
cf : [https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/String/normalize#param%C3%A8tres](url)

Reply:
Nice!  I learned something new today.